build.gradle.kts

```diff
@@ -19,6 +21,8 @@ repositories {
   mavenCentral()
   // OSSRH Snapshots repo
   maven { url = uri("https://s01.oss.sonatype.org/content/repositories/snapshots/") }
+  // for protobuf-gradle-plugin dependencies
+  google()
```

Comment:
?! aren't those on maven central? i never pulled from their maven repo

Reply:
Let me try to build w/o it.

Reply:
Seems to be not needed indeed. Will remove it.